tests/mobile_native/android/test_checkout_android.py

```diff
@@ -1,25 +1,18 @@
-def test_checkout_android(android_emu_driver):
```

Comment:
This file used to contain the react native android test (for empower plant app). All replacements in the file are for regular android tests (toolstore app).